### PR TITLE
Upgrade actions

### PR DESCRIPTION
**Pull request by lldelisle**
Upgrade all used actions to the last major version.

### CODE REVIEW

File: working_fix_syntax
Comment (mvdbeek) [inline working_fix_syntax:1]: I suppose you didn't mean to add submodules here ?

Comment (lldelisle) [reply working_fix_syntax:1]: Argh good catch, sorry for that. I will remove this.

**Comment by lldelisle** [comment]
What is the link between the files in `.github/workflows` and the file in the planemo-ci-action repo https://github.com/galaxyproject/planemo-ci-action/blob/main/.github/workflows/workflows.yaml?

**Review by bgruening — APPROVED** [review]
Don't know the answer to your question. But this PR looks good to me.

**Comment by mvdbeek** [comment]
Attention: deployment skipped!

https://github.com/galaxyproject/iwc/actions/runs/11094477536

**Pull request merged**